internal/server/auth/method/github/server.go

```diff
@@ -185,3 +180,36 @@ func (s *Server) Callback(ctx context.Context, r *auth.CallbackRequest) (*auth.C
 		Authentication: a,
 	}, nil
 }
+
+type githubSimpleOrganization struct {
+	Login string
+}
+
+// api calls Github API, decodes and stores successful response in the value pointed to by v.
+func (s *Server) api(ctx context.Context, token *oauth2.Token, endpoint endpoint, v any) error {
```

Comment:
```suggestion
func api(ctx context.Context, token *oauth2.Token, endpoint endpoint, v any) error {
```

i don't think this need to be a method as it doesn't using anything on `s` it seems. could just be a `func` I think

Reply:
clean it up